src/components/auth/LoginPage.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import Form from 'react-validation/build/form';
+import Input from 'react-validation/build/input';
+import Button from 'react-validation/build/button';
+import { login, logout } from '../../state/auth/auth.actionCreators';
+import './LoginPage.scss';
+
+type Props = {
+    dispatch: Function,
+}
+
+type State = {
+    email: string,
+    password: string,
+    submitted: boolean,
+}
+
+class LoginPage extends Component<Props, State> {
```

Comment:
Since all other components do not include `'Page'` I would just call this one `Login` or `LoginForm`.